pkgs/by-name/ta/taler-exchange/package.nix

```diff
@@ -90,6 +92,13 @@ stdenv.mkDerivation (finalAttrs: {
     popd
   '';
 
+  postFixup = ''
+    for file in taler-config taler-exchange-kyc-kycaid-converter.sh taler-unified-setup.sh; do
```

Comment:
It seems like one more file might need gnunet?

```
rg gnunet /nix/store/y0azc5lsmls7hwm011ldsidy1jqfzc1y-taler-exchange-0.10.2/bin/ -l
/nix/store/y0azc5lsmls7hwm011ldsidy1jqfzc1y-taler-exchange-0.10.2/bin/taler-unified-setup.sh
/nix/store/y0azc5lsmls7hwm011ldsidy1jqfzc1y-taler-exchange-0.10.2/bin/taler-exchange-kyc-persona-converter.sh
/nix/store/y0azc5lsmls7hwm011ldsidy1jqfzc1y-taler-exchange-0.10.2/bin/taler-exchange-kyc-kycaid-converter.sh
/nix/store/y0azc5lsmls7hwm011ldsidy1jqfzc1y-taler-exchange-0.10.2/bin/taler-config
```

Reply:
Don't know how I missed that one, oops 😅